src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    <p className="Person__age">
+      {!person.age && (
+        `I am ${person.age}`
+      )}
+    </p>
+
+    <p className="Person__partner">
+      {!person.isMarried && (
+        `I am not married`
+      )}
+
+      {person.isMarried && (
+        person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`)
+      }
```

Comment:
Move it to variable and write the expression above in terms of a ternary with this variable